sqlite/expr.c

```diff
@@ -5661,6 +5661,21 @@ static char* sqlite3ExprDescribe_inner(
     case TK_FUNCTION: {
       char *ret, *ret2;
       int i;
+      /* dh: not all the functions can be blindely passed remotely! */
+      if(strncasecmp(pExpr->u.zToken, "now", 3) == 0)
+      { 
+        dttz_t dt;
+
+        if(pExpr->x.pList && pExpr->x.pList->nExpr == 1 &&
```

Comment:
The precision of `now()` depends on the default datetime precision. It is millisecond in code however can be changed via LRL.

Please use `DTTZ_TEXT_TO_PREC()` instead. Here is an example:
https://github.com/bloomberg/comdb2/blob/master/sqlite/dttz.c#L352